docs/javascript/v2/advanced-features/playlist.mdx

```diff
@@ -33,7 +33,7 @@ interface HMSPlaylistItem<T> {
     name: string;
     type: HMSPlaylistType;
     /**
-     * the url to play from, local files are not supported currently.
+     * the url to play from
```

Comment:
Let's add - `it can also be a url created from local file eg: URL.createObjectURL(file)`